src/ctia/stores/es/sort.clj

```diff
@@ -18,6 +27,35 @@
       ;; https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_sort_values
       {field-name {:order order}}
 
+      ;; chooses the maximum value in field (a list) based on remappings,
+      ;; and sorts based on that value for each entity. Remappings must
+      ;; be positive integers.
+      ;; eg
+      ;; {:op :remap-list-max
+      ;;  :field-name "tactics"
+      ;;  :remappings {"TA0043" 1
+      ;;               "TA0042" 2}
+      ;;  :sort_order :asc}
+      :remap-list-max
```

Comment:
As this bloc of code seems big, and also, it contains non trivial text (the painless script), perhaps it would be nice to define a function instead.

The function could have the comment as docstring.

Reply:
Combined the `:remap` and `:remap-list-max` cases, and moved comments to doc.